xbmc/interfaces/json-rpc/schema/version.txt

```diff
@@ -1 +1 @@
-JSONRPC_VERSION 10.3.0
+JSONRPC_VERSION 10.3.1
```

Comment:
This makes a non-breaking change - adding fields at end - hence needs to be a bump to 10.4.0

Reply:
Changed. Thanks!